lib/capistrano/tasks/assets.rake

```diff
@@ -62,15 +62,28 @@ namespace :deploy do
       end
     end
 
+    def manifest_path(filename)
+      release_path.join('public', fetch(:assets_prefix), filename)
+    end
+
+    def manifest_and_backup_paths
```

Comment:
@kirs I'm surprised you're seconding @chulkilee 

For some reason, I'm being held to a much higher standard here than the original developers of this gem.  I was the one to suggest some changes to make this code more maintainable, and then I went and did it.

The point of the `manifest_path` method was never to return the `manifest_path`, it was to reduce repetition, and I have asked @chulkilee to propose a more appropriate name for the method in light of that fact, and he has not made such a suggestion.  Instead, he proposed another name (not necessarily a better one IMO) for `backup_path`, a variable that I did not name.

As for @chulkilee's objection to doing 2 things in one method, neither of these 2 things was in a separate method at all before this PR--they were mixed into the code of each of the 2 tasks.  Moving them into a re-usable method is obviously an improvement.  Moving a one-liner (`backup_path = ...`) that is not duplicated anywhere into a separate method is not such an improvement--if someone submitted a PR with only that change, would you consider that enough of a change to be worth merging?